src/unix/network_helper.rs

```diff
@@ -116,3 +126,261 @@ pub(crate) fn get_interface_address() -> Result<InterfaceAddressIterator, String
         }
     }
 }
+
+pub(crate) fn get_interface_ip_networks() -> HashMap<String, HashSet<IpNetwork>> {
+    let mut ifaces: HashMap<String, HashSet<IpNetwork>> = HashMap::new();
+    let mut addrs: MaybeUninit<*mut libc::ifaddrs> = MaybeUninit::uninit();
+
+    // Safety: addrs.as_mut_ptr() is valid, it points to addrs.
+    if unsafe { libc::getifaddrs(addrs.as_mut_ptr()) } != 0 {
+        return ifaces;
```

Comment:
Please add a `sysinfo_debug` call here to display the returned error saying that `getifaddrs` failed.

Reply:
Hope this call will help for debug when need